CIP-governance-metadata-actions/README.md

```diff
@@ -0,0 +1,98 @@
+
+Motivation for tooling:
+By standardizing off-chain metadata formats we facilitate interoperability for tooling which creates and/or renders metadata attached to governance actions.
+This intern promotes the user experience between tooling.
+
+The stake holders for this proposal are all of those users who intend to participate within governance via voting directly.
+Furthermore, governance related indexing and submission based tooling providers. 
+
+## Specification
+Although there are seven types of governance action defined via CIP-1694, we focus this proposal on defining core properties which must be attached to all types.
+We leave room for future standards to refine and specialize further to cater more specific for each type of governance action.
+
+### Extended Vocabulary
+The following properties extend the vocably of CIP-100.
+
+#### Title
+- This should be a sentence which should act as a human readable identifier for the GA
+- It should briefly introduce the GA
+  - i.e; Increase K protocol parameter to 100,000 to increase decentralization of Cardano.
+- The intention for this field is to act as a quick and clear differentiator between governance actions
+
+#### Abstract
+- This should be a short amount of free flow text, limited to ~200 words.
+- This should a short description of the reason behind the GA, so if a problem is being addressed, what why and how
+- This should summarizes the problem and how GA fixes the problem - to help voters gain an understanding the context behind the GA
+
+#### Motivation
+- This should be longer free form text encapsulating context around the problem that the GA is addressing.
+- The content should also include mention of the use cases and stakeholders
+- This field is an opportunity for the author to provide full context to the problem being addressed.
+
+#### Rationale
+- This should be longer free form text discussing how the GA addresses the problem identified in the motivation
+- i.e "by increasing X parameter we increase rewards for SPOs thus encouraging new SPOs to join the network
+- This section should fully justify the changes being made to Cardano
+- This should include what led to particular design decisions
+- It should describe alternate designs considered and related work.
+- The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion.
+- For some GAs this section SHOULD to be very long.
+
+### Versioning
+This proposal should not be versioned, to update this standard a new CIP should be proposed.
+
+## Rationale: how does this CIP achieve its goals?
+- base standard, good for MVP, makes this more straight forward to implement and support for all
+- want to start simple with this proposal - MVG
+- fields based on the CIP process with the specification and most of the header removed
+
+### Open Questions
+- Should fields be optional or compulsory?
```

Comment:
I think it will be pretty difficult to enforce the use of any given field, if someone doesn't provide a field it would be a poor tool that refused to shot the rest of the metadata.

Reply:
No its not.

If we label a field to be compulsory and someone does not do it, then their metadata is not compliant with this CIP and tools may not recognise it

Reply:
I understand what you mean by compulsory, but this would be bad for 2 reasons:
1. would you exclude all metadata not containing a title? If not then which field? 
2. CIP-100 asks that metadata in the wrong JSON format be displayed in a raw format. We have already committed to doing our best to display the metadata that we find.

Reply:
Thats why its an open question :)

Reply:
Depends on which fields the GA has, what is mandatory. It seems to me that in such a serious application, all of the fields should be mandatory where the GA is being described. We will have a different form for each GA, so it seems logical.

As you need a Cardano wallet to submit a GA, your wallet address or stake address is an identifier of the submitter. This is part of the submission, but is it metadata? I would imagine that in the absence of a required 'submitter name' a GA explorer should be able to find a GA by searching for the submitter's wallet or stake address.

I think we could have an optional field for 'friendly name'. This would be good if it's made to be unique. This may not identify anything much, but if a submitter used it multiple times, then the GA explorer would be able to search for it. Also, say if there's loads of GAs in the explorer, if I put in a unique name, I can easily find my proposal.

Reply:
CIP-100 has authors listed, then we extend that with these fields.

Reply:
> Depends on which fields the GA has, what is mandatory. It seems to me that in such a serious application, all of the fields should be mandatory where the GA is being described. 

Does this mean that you would have GovTool forcibly exclude all GA's that had a field missing?  The reason why I think no field should be mandatory, is that I think tooling providers owe their users (the voters) all the information provided and not attempt to curate it in a way that users can't fix.

Reply:
> Does this mean that you would have GovTool forcibly exclude all GA's that had a field missing?

Its up to the tooling makers how they want to deal with it, they can exclude, they can still render, they can show raw JSON. That is more of a product decision.

From an implementation/tech stand point for tools, I can see tools making a reasonable effort to accommodate malformed metadata (a mandatory field may just be left blank). But you must draw the line somewhere (otherwise what is the point of this standard) and governance action authors are incentivised to form metadata correctly IF they want people to be able to read the metadata.